rosidl_generator_dotnet/resource/msg.c.em

```diff
@@ -84,7 +84,10 @@ void @(msg_typename)__write_field_@(member.name)(void * message_handle, @(msg_ty
 @[        end if]@
 }
 @[    else]@
-// TODO: Nested types are not supported
+void * @(msg_typename)__native_get_field_@(member.name)_HANDLE(void * message_handle) {
```

Comment:
For naming consistency with read_field/write_field above:

```suggestion
void * @(msg_typename)__get_field_@(member.name)_HANDLE(void * message_handle) {
```

Reply:
Ok